### PR TITLE
Build two client bundles — ES5 and ES6

### PR DESCRIPTION
**Pull request by plesiecki**

### CODE REVIEW

File: src/common/models/essence/essence.ts [hunk 1]
```diff
@@ -190,8 +190,6 @@ export class Essence extends ImmutableRecord<EssenceValue>(defaultEssence) {
     return timeFilter;
   }
 
-  public visResolve: Resolve;
```
Comment (plesiecki) [inline src/common/models/essence/essence.ts:193]: This cause to `visResolve` to be a `undefined` during runtime.

File: src/server/views.ts [hunk 2]
```diff
@@ -76,6 +76,7 @@ export function mainLayout(options: ViewOptions): string {
   return layout(options, `<div class="app-container"></div>
 <script>var __CONFIG__ = ${JSON.stringify({ version, timekeeper, appSettings: serialize(appSettings) })};</script>
 <script charset="UTF-8" nomodule src="polyfills.js?v=${version}"></script>
+<script charset="UTF-8" async nomodule src="main.es5.js?v=${version}"></script>
```
[the remark this answers is not in the document]
Comment (plesiecki) [reply src/server/views.ts:79]: Actually I forgot to change how modern script is attached. It should be good now.
Basically these are results of separate webpack builds of the same code but with different transpilation level. So no incremental builds here. Legacy builds only for prod though!
Modern browsers utilize scripts attached as `type="module" and will not fetch script attached as `[nomodule]`.
Legacy browsers on the other hand don't understand `[type="module"]` and utilize `[nomodule]` scripts.

File: src/common/models/essence/essence.ts (hunk 1 above)
Comment (adrianmroz-allegro) [reply src/common/models/essence/essence.ts:193]: So probably we will encounter bunch of different weird issues.

[the remark this answers is not in the document]
Comment (plesiecki) [reply src/common/models/essence/essence.ts:193]: It builds ok but it causes `visResolve` to be a `undefined` during runtime. Then `e2e` failed miserably.

Actually I expected more cases like this but it was single one. I'm not sure why that single field out of all fields is re-declare. I assume that parent class declares it and it's superfluous 🤔

**Review by kzadurska — APPROVED** [review]

**Review by adrianmroz-allegro — APPROVED** [review]

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                development: false,
+                useBuiltIns: true,
+            }]
+        ],
+        env: {
+            legacy: {
+                presets: [
+                    ["@babel/preset-env", {
+                        modules: false,
+                        targets: "> 0.25%, last 2 versions, Firefox ESR",
```
Comment (KostkaBrukowa) [inline babel.config.js:22]: we support IE on turnillo?

**Pull request merged**